ui/lib/pki/addon/components/page/pki-configuration-details.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import errorMessage from 'vault/utils/error-message';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+//TYPES
+import RouterService from '@ember/routing/router-service';
+import FlashMessageService from 'vault/services/flash-messages';
+import Store from '@ember-data/store';
+
+interface Args {
+  currentPath: string;
+}
+
+export default class PkiConfigurationDetails extends Component<Args> {
+  @service declare readonly store: Store;
+  @service declare readonly router: RouterService;
+  @service declare readonly flashMessages: FlashMessageService;
+
+  @tracked showDeleteAllIssuers = false;
+
+  @action
+  async deleteAllIssuers() {
+    try {
+      const issuerAdapter = this.store.adapterFor('pki/issuer');
+      issuerAdapter.deleteAllIssuers(this.args.currentPath);
+      this.flashMessages.success('Issuers and keys deleted successfully.');
```

Comment:
Super nit, but I feel like we should add the word "all" here, what do you think? Also, I think we usually word it like `Successfully deleted all issuers and keys` with the operation that just happened is first?

Reply:
Good call! I think it's a good idea to be more specific of what we're deleting!